src/pages/code-of-conduct.mdx

```diff
@@ -3,15 +3,11 @@ title: Code of Conduct
 layout: ../layouts/MarkdownLayout.astro
 ---
 
-import { OrganizationServices } from '~/organization-services';
-
-export const orgInfo = new OrganizationServices().getOrganizationInfo();
-
 ## Anti-Harassment Policy
 
-{orgInfo.name} (GDG) is dedicated to providing a harassment-free and inclusive event experience for everyone regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or other protected category. We do not tolerate harassment of event participants in any form. GDG{" "} {orgInfo.name} takes violations of our policy seriously and will respond appropriately.  
+ GDG is dedicated to providing a harassment-free and inclusive event experience for everyone regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or other protected category. We do not tolerate harassment of event participants in any form. GDG{" "} GDG takes violations of our policy seriously and will respond appropriately.  
```

Comment:
Tried to make this code of conduct generic.

Todo - re-add anonymous reporting.